bin/xspec.bat

```diff
@@ -260,6 +264,15 @@ call :win_reset_options
 call :win_get_options %*
 
 rem
+rem # Schematron
+rem # XQuery
+rem
+if defined SCHEMATRON if defined XQUERY (
+    call :usage "-s and -q are mutually exclusive"
+    exit /b 1
+)
+

```

Comment:
Thanks. I was too lazy to verify it myself, so I pushed Travis/AppVeyor tests for it like [this](https://github.com/AirQuick/xspec/commit/77874e03908a8b10610e0057dc217be853433972).
Can you consider adding something like that to this pull request?